src/helpers/accountId.js

```diff
@@ -0,0 +1,22 @@
+// @flow
+
+import invariant from 'invariant'
+
+type Params = {
+  type: string,
+  version: string,
+  xpub: string,
+  walletName: string,
+}
+
+export function encode({ type, version, xpub, walletName }: Params) {
+  return `${type}:${version}:${xpub}:${walletName}`
```

Comment:
Agree :+1: 

Additionally, it would be useful to have an `export to file <-> import from file` tool (usable from the CrashScreen) to ease support. Can be interesting to allow detailed exports:

```
Export data:
  [x] accounts
    [x] anonymize (randomize names, etc.)
    [ ] include operations
  [ ] settings
```

Reply:
(interesting idea! yeah definitely. & for logs we need a proper logger system on all user actions / redux actions so we can debug appropriately ;) I'll spec out something this week around this subject. it's quite easy to add, i was thinking dom elements could have a "role" attr that expose them to this system & we track&log all onclick. whatever system we can figure out that is enough debuggable but also easy to add (we should not have to console.log at each single onClick is what i mean) )

Reply:
going forward on that subject, being able to retrieve apdu calls (and other precise stuff) occuring on a crash could be very useful.